database_schemas/cassandra/benchflow.cql

```diff
@@ -394,7 +400,18 @@ CREATE TABLE exp_faban_ops_custom_stats (
   faban_stat_mode list<double>,
   faban_stat_mode_freq bigint,
   faban_stat_sd double,
-  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name, faban_op_name, faban_op_stat_name)
+  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name, faban_stat_name)
+);
+
+CREATE TABLE exp_faban_ops_custom_stats_composite (
```

Comment:
What do you save in this table? The stat on top of the `target=absolute` metrics? If yes, why do you call them `_stat` and why we just have `min` and `max` also here since the starting data are absolute numbers and not statistics?